server/views/index.html

```diff
@@ -162,7 +162,7 @@
                                  </md-menu-item>
                                  <md-menu-divider></md-menu-divider>
                                  <md-menu-item>
-                                    <md-button ng-click="ctrl.menuAction('file.save', $event)">
+                                    <md-button ng-disabled="!shouldSave" ng-click="ctrl.menuAction('file.save', $event)">
```

Comment:
Situations where the files can't be saved are:
   1 - The FBP file is inside the demo folder.
   2 - The FBP does not belong to any project and the file wasn't created before. (This happens when you start coding without importing or creating a new project).

Some improvements for the number 2 were suggested here: https://github.com/solettaproject/soletta-dev-app/issues/85